src/main/java/io/edpn/backend/util/Topic.java

```diff
@@ -0,0 +1,42 @@
+package io.edpn.backend.util;
+
+public interface Topic {
+
+    enum Request {
```

Comment:
is "request" the correct naming?

EDDN is not really a request, but an update, in my opinion.

Reply:
this is for the inter module communication.

these are the topics where our own modules will put a message to request data from another module